### DISCUSSION TITLE
Visualising Multiple Episodes

### DISCUSSION THREAD

**Discussion by cemlyn007** [google/brax#226] [Ideas]
<img width="1194" alt="Merged ant episodes" src="https://user-images.githubusercontent.com/49565105/191835714-99ecafd0-d0d2-4816-9f87-f200e97eaa40.png">

Hey 👋,

In the above picture, I am visualising multiple episodes as if it was a single episode so that I can see how my agent performs as it went through training. Is this something you would like to see more of?

I take a single environment, multiple episodes and merge all of the episodes into a single episode by merging the QPs together and duplicating the environment `brax.Config`.

Note that my implementation has been tested with the `io.image` and `io.html` modules.

Attached is a link to a html version: [view](https://drive.google.com/file/d/1wkF_y6QY05BSUqM2ecutJqNGBDT7RXza/view?usp=sharing).

**Comment by erikfrey** [comment]
This is awesome!  We are going to soon release a significant update to brax along with some new visualizer code, but it doesn't include this functionality.  I'm going to leave this discussion open and check back in with you after the release.  We'd love to support something like this.
